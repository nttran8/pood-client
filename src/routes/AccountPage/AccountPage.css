.AccountPage {
  padding-top: 15px;
}

.AccountPage > h2 {
  color: var(--color-comp-w);
}

@media only screen and (min-width: 567px) {
  .AccountPage > h2 {
    color: var(--color-triad-b);
  }
}
@media only screen and (min-width: 768px) {
  .AccountPage {
    padding-top: 30px;
  }

  .AccountPage > h2 {
    font-size: 30px;
  }
}