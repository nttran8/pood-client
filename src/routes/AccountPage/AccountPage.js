// Library
import React from "react";

// Component
import Account from "../../components/Account/Account";

// Style
import "./AccountPage.css";

export default function AccountPage() {
  return (
    <section className="AccountPage">
      <h2>Account Setting</h2>
      <Account />
    </section>
  );
}
