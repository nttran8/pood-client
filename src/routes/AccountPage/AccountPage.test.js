import React from "react";
import TestRenderer from "react-test-renderer";
import AccountPage from "./AccountPage";

it(`Renders AccountPage UI as expected`, () => {
  const tree = TestRenderer.create(<AccountPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
