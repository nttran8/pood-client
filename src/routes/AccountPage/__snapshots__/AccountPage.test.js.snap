// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<section
  className="AccountPage"
>
  <h2>
    Account Setting
  </h2>
  <form
    className="AccountForm"
    onSubmit={[Function]}
  >
    <div
      role="alert"
    />
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="username"
      >
        Username
      </label>
      <input
        className="input acc-username"
        id="username"
        onClick={[Function]}
        readOnly={true}
      />
    </div>
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="fullname"
      >
        Fullname
      </label>
      <input
        className="input fullname"
        defaultValue=""
        id="fullname"
      />
    </div>
    <div
      className="dataBox"
    >
      <label
        className="accountLabel"
      >
        Gender
      </label>
      <div
        className="genderSelection"
      >
        <input
          checked={false}
          className="input "
          id="female"
          name="gender"
          onChange={[Function]}
          type="radio"
          value="female"
        />
        <label
          htmlFor="female"
        >
          female
        </label>
        <input
          checked={false}
          className="input "
          id="male"
          name="gender"
          onChange={[Function]}
          type="radio"
          value="male"
        />
        <label
          htmlFor="male"
        >
          male
        </label>
      </div>
    </div>
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="email"
      >
        Email*
      </label>
      <input
        className="input email"
        id="email"
        required={true}
      />
    </div>
    <button
      className="button accountButton"
      type="submit"
    >
      Update
    </button>
  </form>
</section>
`;

exports[`Renders AccountPage UI as expected 1`] = `
<section
  className="AccountPage"
>
  <h2>
    Account Setting
  </h2>
  <form
    className="AccountForm"
    onSubmit={[Function]}
  >
    <div
      role="alert"
    />
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="username"
      >
        Username
      </label>
      <input
        className="input acc-username"
        id="username"
        onClick={[Function]}
        readOnly={true}
      />
    </div>
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="fullname"
      >
        Fullname
      </label>
      <input
        className="input fullname"
        defaultValue=""
        id="fullname"
      />
    </div>
    <div
      className="dataBox"
    >
      <label
        className="accountLabel"
      >
        Gender
      </label>
      <div
        className="genderSelection"
      >
        <input
          checked={false}
          className="input "
          id="female"
          name="gender"
          onChange={[Function]}
          type="radio"
          value="female"
        />
        <label
          htmlFor="female"
        >
          female
        </label>
        <input
          checked={false}
          className="input "
          id="male"
          name="gender"
          onChange={[Function]}
          type="radio"
          value="male"
        />
        <label
          htmlFor="male"
        >
          male
        </label>
      </div>
    </div>
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="email"
      >
        Email*
      </label>
      <input
        className="input email"
        id="email"
        required={true}
      />
    </div>
    <button
      className="button accountButton"
      type="submit"
    >
      Update
    </button>
  </form>
</section>
`;

exports[`Renders EndPage UI as expected 1`] = `
<section
  className="AccountPage"
>
  <h2>
    Account Setting
  </h2>
  <form
    className="AccountForm"
    onSubmit={[Function]}
  >
    <div
      role="alert"
    />
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="username"
      >
        Username
      </label>
      <input
        className="input acc-username"
        id="username"
        onClick={[Function]}
        readOnly={true}
      />
    </div>
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="fullname"
      >
        Fullname
      </label>
      <input
        className="input fullname"
        defaultValue=""
        id="fullname"
      />
    </div>
    <div
      className="dataBox"
    >
      <label
        className="accountLabel"
      >
        Gender
      </label>
      <div
        className="genderSelection"
      >
        <input
          checked={false}
          className="input "
          id="female"
          name="gender"
          onChange={[Function]}
          type="radio"
          value="female"
        />
        <label
          htmlFor="female"
        >
          female
        </label>
        <input
          checked={false}
          className="input "
          id="male"
          name="gender"
          onChange={[Function]}
          type="radio"
          value="male"
        />
        <label
          htmlFor="male"
        >
          male
        </label>
      </div>
    </div>
    <div
      className="inputForm dataBox"
    >
      <label
        className="accountLabel"
        htmlFor="email"
      >
        Email*
      </label>
      <input
        className="input email"
        id="email"
        required={true}
      />
    </div>
    <button
      className="button accountButton"
      type="submit"
    >
      Update
    </button>
  </form>
</section>
`;
